app.js

```diff
@@ -204,6 +226,7 @@ function createChart () {
     }
   };
 
+  //linter errors exist because referencing 'chartJS' library
   var junkChart = new Chart(context, {
```

Comment:
Since you are not doing anything with this Chart after creating it, you do not need to store it in a variable.